docs/logs/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,128 @@
+# Getting Started with OpenTelemetry .NET Logs in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new web -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.Console](../../../src/OpenTelemetry.Exporter.Console/README.md)
+and
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Extensions.Hosting
+```
+
+Update the `Program.cs` file with the code from [Program.cs](./Program.cs).
+
+Run the application (using `dotnet run`) and then browse to the URL shown in the
+console for your application (e.g. `http://localhost:5000`). You should see the
```

Comment:
since we removed built-in ConsoleProvider, do we still see the url in console?

Reply:
Yes, from the OTel console exporter.